CI/flatpak/com.obsproject.Studio.json

```diff
@@ -186,8 +186,7 @@
         {
           "type": "git",
           "url": "https://aomedia.googlesource.com/aom.git",
-          "tag": "v3.2.0",
-          "commit": "287164de79516c25c8c84fd544f67752c170082a"
```

Comment:
Hm, is there really no tagged release we can use? If not, sad :disappointed: 

I see many commits newer than this in the `main` branch of `aom`, perhaps you want to select a more recent one?

Reply:
For obsproject/obs-deps#90, I simply used the commit hash I was given by others pushing for updates and what was deployed with 27.2.0-beta4.  If there's an argument to use something different, please voice it there.

Reply:
Not really, but it would have helped if @tytan652 mentioned in the commit message that this is the same commit used by obs-deps. @tytan652 could you please add this information to the commit?